std/experimental/normint.d

```diff
@@ -0,0 +1,876 @@
+// Written in the D programming language.
+
+/**
+This module implements support for normalized integers.
+
+Authors:    Manu Evans
+Copyright:  Copyright (c) 2015, Manu Evans.
+License:    $(WEB boost.org/LICENSE_1_0.txt, Boost License 1.0)
+Source:     $(PHOBOSSRC std/experimental/_normint.d)
+*/
+module std.experimental.normint;
+
+import std.traits : isIntegral, isSigned, isUnsigned, isFloatingPoint, Unsigned;
+static import std.algorithm;
+
+import std.experimental.color : FloatTypeFor;
```

Comment:
This file is external to color itself, it shouldn't be importing color directly.
Is there some way to remove it?